common.py

```diff
@@ -0,0 +1,110 @@
+import csv
+import os
+import urllib.request
+
+from IPython import embed
+
+
+class FileLoaderMixin():
+
+    def load_file(self, file_path):
+
+        # make sure that we only try to load supported file types
+        supported_file_types = {'.csv', '.txt'}
+        current_file_type = file_path[file_path.rfind('.'):]
+        if not current_file_type in supported_file_types:
+            raise ValueError("The FileLoaderMixin currently supports {supported_file_types} but not "
+                             "{current_file_type}.")
+
+        # check if we are working locally and in the correct directory
+        # __file__ is only available if executed from a file but not from an ipython shell or notebook
+        # in those cases, the file has to be loaded remotely from github.
+        try:
+            local_path = os.path.abspath(os.path.dirname(__file__))
+            is_local = True
+            if not local_path.endswith('/gender_novels'):
+                is_local = False
+                print(f"WARNING: The FileLoaderMixin should be placed in the main path of the gender_novels project."
+                      f"It's currently in {local_path}. Until the Mixin is in the correct path, files are loaded " \
+                      f"remotely.")
+        except NameError:
+            is_local = False
+
+        if is_local:
+            print(f'loading {file_path} locally.')
```

Comment:
for debugging statements, we should be in general use this paradigm:

```
if DEBUG: 
    print(...debugmsg...)
```

Reply:
for now, that'd be great -- we just want to be able to turn off all debug statements in one fell swoop.  Logging levels are great too eventually.  The music21 solution is a bit bulky, agreed.